app/views/projects/blob/_replace.html.haml

```diff
@@ -0,0 +1,27 @@
+#modal-replace-blob.modal
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %a.close{href: "#", "data-dismiss" => "modal"} ×
+        %h3.page-title Replace #{@blob.name}
+        %p.light
+          From branch
+          %strong= @ref
+      .modal-body
+        = form_tag namespace_project_update_blob_path(@project.namespace, @project, @id), method: :put, id: 'blobFileDropzone', class: 'dropzone form-horizontal' do
+          %br
+            .form-group
+              .col-sm-offset-2.col-sm-10
+                .dropzone-previews
+                  %p.dz-message You can drop a file here or click to upload.
+                  .fallback
+                    = file_field_tag "file", multiple: true, id: "replace_file"
```

Comment:
If a user somehow gets to this modal and disables JavaScript, this fallback will be used. Will the controller still work in this mode?